ethcore/src/spec/spec.rs

```diff
@@ -111,6 +111,8 @@ pub struct CommonParams {
 	pub remove_dust_contracts: bool,
 	/// Wasm support
 	pub wasm: bool,
```

Comment:
Why is that needed? Seems that it's just `wasmTransition: 0`. We already did breaking changes to spec files in the past, we should just highlight that in the release notes.